skimage/draw/draw.py

```diff
@@ -8,18 +8,33 @@
                     _bezier_curve)
 
 
-def _ellipse_in_shape(shape, center, radiuses):
-    """Generate coordinates of points within ellipse bounded by shape."""
+def _ellipse_in_shape(shape, center, radiuses, orientation=0.):
+    """ Generate coordinates of points within ellipse bounded by shape.
+    see: http://www.maa.org/external_archive/joma/Volume8/Kalman/General.html
```

Comment:
Put the reference in a `References` section at the bottom of the docstring - look elsewhere in the package for the proper Sphinx formatting for this.